src/js/control-bar/audio-track-controls/audio-track-button.js

```diff
@@ -0,0 +1,59 @@
+import MenuButton from '../../menu/menu-button.js';
+import Component from '../../component.js';
+import * as Fn from '../../utils/fn.js';
+import AudioTrackMenuItem from './audio-track-menu-item.js';
+
+class AudioTrackButton extends MenuButton {
```

Comment:
what about having this button extend `TrackButton` and refactor `TextTrackButton` into a `TrackButton`? it already does the `removetrack` `addtracker` listening.